R/mod_export.R

```diff
@@ -10,17 +10,24 @@
 mod_export_ui <- function(id) {
   ns <- NS(id)
   tagList(
-    actionButton(ns("export"), "Export")
+    actionButton(ns("export"), "Export", disabled = TRUE)
   )
 }
 
 #' export Server Functions
 #'
 #' @noRd
-mod_export_server <- function(id, r) {
+mod_export_server <- function(id, r, configuration_path) {
```

Comment:
configuration_path is already part of the `r` object. Use it from here instead of adding a new argument.